modules/nodes-pods-vertical-autoscaler-about.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-vertical-autoscaler.adoc
+
+[id="nodes-pods-vertical-autoscaler-about_{context}"]
+= About the Vertical Pod Autoscaler
+
+A Vertical Pod Autoscaler (VPA) uses a Mutating Admission Webhook to automatically compute historic and current CPU and memory usage for the containers in Pods associated with specific Pod controllers, such as Deployments, StatefulSets, Jobs, DaemonSets, ReplicaSets, or ReplicationControllers. You can configure a VPA to use the data it learns to automatically re-deploy Pods with optimized resource limits and requests to ensure that these Pods are operating efficiently at all times. When re-deploying Pods, a VPA honors any Pod Disruption Budget set for your applications. If you do not want a VPA to automatically re-deploy Pods, you can use this resource information to manually update the Pods as needed.
```

Comment:
I don't think that the webhook should be mentioned here.  Maybe instead, it could say:

> The Vertical Pod Autoscaler (VPA) monitors pod metrics to automatically ...

Looking at upstream docs for HPA, they do a nice job of explaining the relationship between the controllers and the API object (which they call a resource). See https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/ for what I'm talking about. I like what they say here:

> The Horizontal Pod Autoscaler is implemented as a Kubernetes API resource and a controller. The resource determines the behavior of the controller. The controller periodically adjusts the number of replicas in a replication controller or deployment to match the observed average CPU utilization to the target specified by user.

So they define the HPA as both things, which I think makes a lot of sense. So maybe we should look at the VPA similarly, with the VPA being the combination of controllers and the object together.

Reply:
@joelsmith 
_The Vertical Pod Autoscaler Operator (VPA) is implemented as an API resource and a custom resource (CR) object. The CR determines the actions the VPA should take with the Pods associated with a specific workload object, such as a Daemonset, ReplicationController, and so forth._ 

Also, since the `auto` and `recreate` modes essentially do the same thing currently, does it make sense to list both options? Especially where the Kubernetes doc suggests that `recreate` **should be used rarely**.  Should we remove the `recreate` option?
@rphillips